sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -192,7 +192,7 @@ trait CheckAnalysis extends PredicateHelper with LookupCatalog with QueryErrorsB
     )
   }
 
-  private def isContainsUnsupportedLCA(e: Expression, operator: LogicalPlan): Boolean = {
+  private def containsUnsupportedLCA(e: Expression, operator: LogicalPlan): Boolean = {
```

Comment:
Let's avoid having such typo fixes ... .this is really something we can easily fix when we happen to touch those codes later...

Reply:
bq. when we happen to touch those codes later
Agreed